dbms/programs/client/Client.cpp

```diff
@@ -87,12 +91,79 @@ namespace ErrorCodes
     extern const int CLIENT_OUTPUT_FORMAT_SPECIFIED;
 }
 
-
 class Client : public Poco::Util::Application
 {
 public:
     Client() {}
 
+    void init_suggestions(Completion::TSTNode &node)
+    {
+        QUERYPART *qP = queryParts;
+        while (qP->name) {
+            node.add_word(qP->name);
+            qP++;
+        }
+
+        std::vector<Block> blocks;
+
+        //preload all functions
+        sendQuery("SELECT name FROM system.functions", blocks);
```

Comment:
Aggregation function can also have suffixes

Reply:
Wont it be too much repetition and will pollute suggestions? For that reason i didn't want to include combinators, but can add them regardless.

Reply:
May be you right.